regler/src/main/kotlin/no/nav/pleiepengerbarn/uttak/regler/UttaksplanMerger.kt

```diff
@@ -6,10 +6,11 @@ import no.nav.pleiepengerbarn.uttak.kontrakter.*
 
 object UttaksplanMerger {
 
-    fun slåSammenUttaksplaner(forrigeUttaksplan: Uttaksplan, nyUttaksplan: Uttaksplan): Uttaksplan {
+    fun slåSammenUttaksplaner(forrigeUttaksplan: Uttaksplan, nyUttaksplan: Uttaksplan, trukketUttak: List<LukketPeriode>): Uttaksplan {
         val timelineForrigeUttaksplan = lagTimeline(forrigeUttaksplan)
+        val timelineForrigeUttaksplanMinusTrukketUttak = fjernTrukketUttak(timelineForrigeUttaksplan, trukketUttak)
         val timelineNyUttaksplan = lagTimeline(nyUttaksplan)
-        return  lagSammenslåttUttaksplan(timelineForrigeUttaksplan, timelineNyUttaksplan)
+        return  lagSammenslåttUttaksplan(timelineForrigeUttaksplanMinusTrukketUttak, timelineNyUttaksplan)
```

Comment:
Flisespikkeri, men flere av return-statementsene har to mellomrom etter seg fremfor kun én.

Reply:
Fikset